### ISSUE TITLE
No 120hz support anymore

### ISSUE THREAD

**Issue by aykayfpv** [maxrave-dev/SimpMusic#478]
The app doesn't support 120hz anymore. Only 60hz. Also, it isn't as smooth anymore but that can be because of the missing refreshrate support.

**Comment by CrazyWolf13** [comment]
Hi, can you confirm 120Hz does still not work with version 0.2.19?

**Comment by aykayfpv** [comment]
Yeah, that problem is already outdated. Back then, it was a performance issue that slowed the app's refresh rate. In 0.2.19, everything is smooth again, even at 120Hz.
